### ISSUE TITLE
Use this card with a simple bluetooth dongle instead of an ESP32 device

### ISSUE THREAD

**Issue by maxime1992** [IhorSyerkov/linak-desk-card#18]
Hello :wave: 

In case it can be of any use to others, I wanted to get this card working with a simple bluetooth dongle instead of having to buy a separate, ESP32 device.

I've created the following repo: https://github.com/maxime1992/linak-2-mqtt Everything is explained in the readme :+1: 

_(feel free to close that issue anytime :+1: just thought it may be useful for other people!)_

**Comment by IhorSyerkov** [comment]
Hello. I'll take a look how it can be adopted :)

**Comment by IhorSyerkov** [comment]
Looks like you make you component work with my card. If so, and nothing else required from my side I can at lest add link to your project in `README.md`. Thanks 😄

**Comment by maxime1992** [comment]
Yes I'm indeed using your card _(wouldn't have spammed you otherwise :angel:)_.

The only thing that's not working is the up and down arrow but it should be simple enough for someone else to do that binding and raise a PR if they feel like it. All I was interested in was the sit and up positions so these 2 are working for sure and the current height is displayed correctly as well.

Good idea for the readme :)
